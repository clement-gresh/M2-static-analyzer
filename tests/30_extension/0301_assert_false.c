{
  int x;
  x = rand(0,10);
  if(x < 2) { x = x -10; }
  assert (x < 5);
  print(x);
}
