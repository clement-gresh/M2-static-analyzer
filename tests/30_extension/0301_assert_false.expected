tests/30_extension/0301_assert_false.c:5.2-17: ERROR: assertion failure
tests/30_extension/0301_assert_false.c:6.2-11: { [ x in [-10;-9] ] }, { [ x in [2;4] ] }
analysis ended